sympy/series/tests/test_limits.py

```diff
@@ -308,6 +308,15 @@ def test_series_AccumBounds():
     assert limit(e, x, oo) == AccumBounds(0, oo)
 
 
+def test_pull_request_22491():
```

Comment:
These tests can be added to one of the `test_basic` and `test_log` functions - no need to make a new function. A reference to this PR can be put in the comments under those functions.